src/Intl/Grapheme/Grapheme.php

```diff
@@ -191,6 +192,38 @@ public static function grapheme_strstr($s, $needle, $beforeNeedle = false)
         return mb_strstr($s, $needle, $beforeNeedle, 'UTF-8');
     }
 
+    public static function grapheme_str_split($s, $len = 1) {
+        if ($len < 0 || $len > 1073741823) {
+            if (80000 > \PHP_VERSION_ID) {
+                return false;
+            }
+
+            throw new \ValueError('grapheme_str_split(): Argument #2 ($length) must be greater than 0 and less than or equal to 1073741823.');
+        }
+
+        if ($s === '') {
+            return [];
+        }
+
+        preg_match_all('/\X/u', $s, $matches);
+
+        if (empty($matches[0])) {
+            return false;
+        }
+
+        if ($len === 1) {
+            return $matches[0];
+        }
+
+        $chunks = array_chunk($matches[0], $len);
+
+        array_walk($chunks, static function(&$value) {
```

Comment:
I'm pretty sure using a foreach is going to be faster here: not need for one function call per iteration